base/atomics.jl

```diff
@@ -88,10 +108,21 @@ function atomic_cas! end
 
 Atomically exchange the value in `x`
 
-Atomically exchanges the value in `x` with `newval`. Returns the old
+Atomically exchanges the value in `x` with `newval`. Returns the old (!)
```

Comment:
Hm you're right that is used in a bunch of these, without explanation. Couldn't we just emphasize "old" with italic or bold?